lib/server/sitemap.js

```diff
@@ -8,31 +8,25 @@
 const fs = require('fs-extra');
 
 const glob = require('glob');
+
 const CWD = process.cwd();
 
 const sitemap = require('sitemap');
 const utils = require('../core/utils');
 
 const siteConfig = require(CWD + '/siteConfig.js');
 
-/****************************************************************************/
-
-let readMetadata;
-let Metadata;
-let MetadataBlog;
+const readMetadata = require('./readMetadata.js');
+const Metadata = require('../core/metadata.js');
+const MetadataBlog = require('../core/MetadataBlog.js');
```

Comment:
Need to call 
```js
readMetadata.generateMetadataBlog();
```

Because metadatablog might not exist when requiring (test are failing as well)

Similarly to docs